executor/src/witgen/bit_constraints.rs

```diff
@@ -136,7 +138,7 @@ pub fn determine_global_constraints<'a>(
     for identity in identities {
         let remove;
         (known_constraints, remove) =
-            propagate_constraints(fixed_data, known_constraints, identity, &full_span);
```

Comment:
Where's `fixed_data` now?

Reply:
Not needed any more. Using `&PolynomialReference` as keys removes the need to access `fixed_data` as lookup table.